test/evergreen.yml

```diff
@@ -1638,6 +1662,105 @@ tasks:
               cp -rf WT_TEST WT_TEST_$file
             done
 
+  - name: check-test-long-1
+    tags: ["check-long-test"]
```

Comment:
Why bothering with a tag setting here?

Reply:
Reminiscence of time when I actually started with bunch of tasks. Fused together tasks but didn't remember to remove the tags. Will fix it.